inc/template-tags.php

```diff
@@ -116,51 +116,72 @@ function primer_the_page_title() {
 /**
  * Display navigation to next/previous set of posts when applicable.
  *
- * @global WP_Query $wp_query
- * @since  1.0.0
+ * @deprecated NEXT No longer supported as the pagination funciton. See primer_pagination().
  */
-function primer_paging_nav() {
-
-	global $wp_query;
+function primer_paging_nav( $args = array() ) {
 
-	if ( ! isset( $wp_query->max_num_pages ) || $wp_query->max_num_pages < 2 ) {
+	if ( ! current_user_can( 'edit_themes' ) ) {
 
 		return;
 
 	}
 
-	?>
-	<nav class="navigation paging-navigation">
+	trigger_error( __( 'primer_paging_nav() has been deprecated since NEXT. Use primer_pagination() instead.', 'primer' ) );
```

Comment:
@EvanHerman The Grunt task won't recognize the use of `NEXT` like this. Maybe in addition to replacing `@since NEXT` we should also add support for `@NEXT`?